doc/contrib/xential.rst

```diff
@@ -0,0 +1,45 @@
+Xential
+=======
+
+Integration
+-----------
+
+BPTL can automate document creation using Xential templates.
+
+The `bptl.work_units.xential.tasks.start_xential_template` work-unit needs to know the UUID of the template to
+use and whether to build the document interactively (the user fills the empty fields in the template) or
+'silently'. In the latter case, values to fill the template fields also need to be provided to the work-unit.
```

Comment:
for silent, they _need_ to be provided indeed, for interactive, they _may_ be provided.